assets/src/dashboard/app/views/templates/index.js

```diff
@@ -151,6 +151,8 @@ function TemplatesGallery() {
             centerActionLabel={__('View details', 'web-stories')}
             bottomActionLabel={__('Use template', 'web-stories')}
             updateStory={() => {}}
+            duplicateStory={() => {}}
+            trashStory={() => {}}
```

Comment:
will templates ever have these actions? If not maybe we should set them as optional and check they exist before setting them in the menu?  This sounds like it's out of scope, just a question!

Reply:
Yeah I'm not sure 🤔 I just wanted the prop errors to go away.  I believe Carlos is working on (or going to work on) updating the template gallery UI to remove title and modify date from the cards, so I'mma let him figure out how best to handle these in the long term (I think they'll just be going away, they may not need a context menu like the normal stories 🤔).